mypy/plugins/default.py

```diff
@@ -20,16 +20,26 @@
 class DefaultPlugin(Plugin):
     """Type checker plugin that is enabled by default."""
 
+    def get_additional_deps(self, file: MypyFile) -> List[Tuple[int, str, int]]:
+        if self.python_version >= (3, 8):
+            # Add module needed for anonymous TypedDict (used to support dataclasses.asdict)
+            return [(10, "typing", -1)]
```

Comment:
Why this is needed? I think `typing` module is always loaded. It may be needed to add only dependency on `typing_extensions`, which is available on all versions.

Reply:
It seems adding that dependency on `typing_extensions` means a lot of test fixtures need updating (it's the `tuple` builtin missing, which I believe is transitively imported from `typing_extensions`).